### ISSUE TITLE
As a developer I would like a centralized place in the wiki for information on the database.

### ISSUE THREAD

**Issue by jkrod** [aau-giraf/wiki#200]
**Is your feature request related to a problem? Please describe.**
First off, this is was prompted by issue #189 and is related to #124 .

The problem is: In the wiki the information about the database is frustrating to find for a developer, as it is not centralized in a section called database but rather spread out across the Server Administration and Rest Api Development tabs in the Development section of the database.

I am not sure if I have located everywhere, but at least it is in the Database subsection (Which doesnt contain much of the database) of Rest Api Development and the Backend Arkitektur subsection (which contains the diagram of the database). Depending on what is written in a new database section it could also be related to Models and DTOs section, so be sure to check all sections when solving this issue. For visual locations see screendump below:

![Udklip](https://user-images.githubusercontent.com/34282060/81043629-e5dcd800-8eb2-11ea-8610-5af8d6c21eff.JPG)

**Describe the solution you'd like**
I would like a separate parent section for Database in the Development tab of the wiki, such that info on the database is more centralized. It also needs to be updated with changes made to the database and a new diagram. 

Also currently there is not a lot of explanation of the diagram, which could be better. The current explanation of the figure below is old and missing explanations and doesn't cover everything. For example one could explain that the small teal dots to the left of the attributes means those attributes are nonnull.

![image](https://user-images.githubusercontent.com/34282060/81044191-25f08a80-8eb4-11ea-88c8-c12625e54ffc.png)

Also keep cross-references to the database in mind, when moving the sections to the centralized one. 

I recommend either having somewhere in the new Database section a cross-reference to the database section under Server Administration > Docker swarm > Database or move that section under the centralized one as well, whatever seems most appropriate. 

**######################################################################**

**TLDR:** Database should have a separate section under development in the wiki and be cross-referenced at relevant places instead of having it spread out. It should be better explained, as well as updated according to changes made.

**Comment by mads-bo** [comment]
Vær opmærksom på at der i bunden af de fleste sider er et link til hvad man skal læse som det næste, som er lavet sidste år. Det kan måske hjælpe til hvordan det skal hænge sammen, men husk at være opmærksom på at det der ikke bliver påvirket af dette issues ændringer, stadig skal hænge sammen.

**Comment by LivHolm** [comment]
Det her issue er blevet fikset i forbindelse med dette issue #267
